web/source/text/outputTarget.ts

```diff
@@ -26,28 +26,35 @@ namespace com.keyman.text {
     readonly token: number;
     readonly keystroke: KeyEvent;
     readonly transform: Transform;
+    readonly alternates: Alternate[];
     readonly preInput: Mock;
 
-    // While not presently needed, there's a chance we'll want to have this information tracked
-    // for developments later down the line.  May as well resolve the logic now.
-    readonly removedDks: Deadkey[];
-    readonly insertedDks: Deadkey[];
+    // // While not presently needed, there's a chance we'll want to have this information tracked
```

Comment:
Again, I'd put this in the `delete deadkey mutation` commit and revert the commit later.